### ISSUE TITLE
A11y_Model Builder_Select Scenario_Info and Relationships : Narrator/NVDA is not announcing the category/type of the scenario along with scenario info.

### ISSUE THREAD

**Issue by manaswita-chougule** [dotnet/machinelearning-modelbuilder#2678]
### GitHub Tags
#WCAG1.3.1; #A11ySev2; #A11yTCS; #A11yWCAG2.1; #DesktopApp; #Win11; #Visual Studio Client; #SH_ModelBuilder_Win32_May2023; #Info and Relationships; #Win32; #Narrator; #NVDA;

### Actual Result
Narrator/NVDA is not announcing the category/type along with scenario info. When narrator/NVDA focus lands on Data Classification scenario tile, it is only announcing scenario information and not the category/type of the scenario as "Tabular".

### Expected Result
Narrator/NVDA should announce the category/type of the scenario along with scenario info. When narrator/NVDA focus lands on Data Classification scenario tile, it should announce scenario information and the category/type of the scenario as "Tabular".

### Environment
Application Name: Model Builder
Microsoft Visual Studio Enterprise 2022 (64-bit) - Current
Version 17.6.1
Microsoft Windows 11 Enterprise Build 22621.1702
Narrator
NVDA : Version 2023.1

### Repro Steps

1. Launch the Visual Studio with at least one of the .NET workloads.
2. Tab till enable ML.NET Model Builder (Tools -> Options -> Environment -> Preview Features) and select it.
3. Create a new .NET project or open an existing .NET project.
4. Open context menu (Right-click) of the project in the solution explorer.
5. Tab till Add and then tab till Machine Learning Model and press enter.
6. Model Builder dialog will open. 
7. Turn on Narrator/NVDA.
8. Tab till data classification tile.
9. Observe that when narrator/NVDA focus lands on Data Classification scenario tile, it is only announcing scenario information and not the category/type of the scenario as "Tabular".

### User Impact
Screen reader users will not able to get the information about the category/type of the scenario if it is not being announced by screen reader. 

### Attachment
Narrator
![When narrator focus lands on Data Classification scenario tile, it is only announcing scenario information and not the section heading as Tabular](https://github.com/dotnet/machinelearning-modelbuilder/assets/91586101/d2063f60-1167-4ce9-acc6-2728be03317d)

NVDA
![When NVDA focus lands on Data Classification scenario tile, it is only announcing scenario information and not the section heading as Tabular](https://github.com/dotnet/machinelearning-modelbuilder/assets/91586101/5d058451-b92f-401d-9498-ac6dafb81f31)

**Comment by zewditu** [comment]
"Tabular" is not an interactive element are we supposed to make the narrator narrate it?

**Comment by manaswita-chougule** [comment]
#Regressed:06-02-23;

@zewditu As the "Tabular", "Natural Language Processing" and "Computer Vision" is visually appearing as Label for the group of scenario types, so narrator should announce it when any of the scenario type is focused. Thanks!

**Comment by zewditu** [comment]
@manaswita-chougule , ready to validate by Model Builder version 16.17.0.2332602.

**Comment by manaswita-chougule** [comment]
#Regressed:06-29-23; #Closed;
Verified the issue in below environment:
Visual Studio : Microsoft Visual Studio Enterprise 2022 (64-bit) - Current Version 17.6.3
Model Builder : Version 17.17.0.2332602

The issue is fixed, hence closing the bug. Please find attachment below for reference. Thanks!
Narrator:
![Narrator announcing the category along with scenario info](https://github.com/dotnet/machinelearning-modelbuilder/assets/91586101/0eb6a40e-3f39-48a6-a4ea-0dc64b817cb4)

NVDA:
![NVDA announcing the category along with scenario info](https://github.com/dotnet/machinelearning-modelbuilder/assets/91586101/b68f5d10-429a-40d5-9aa8-ae5d1f0661b6)
